client.go

```diff
@@ -68,25 +73,20 @@ func getDockerClient() DockerClient {
 	return dockerClient
 }
 
-// Run the given command in a container based on the given image. The given
-// image string is just the ID of said image. It returns true if the command
-// was successful, false otherwise.
-func runCommandInContainer(img string, cmd []string) bool {
+// Looks for the specified command inside of a Docker image.
+// The given image string is just the ID of said image.
+// It returns true if the command was successful, false otherwise.
+func checkCommandInImage(img string, cmd string) bool {
```

Comment:
However, you only use `cmd` for passing it to `createContainer` which needs a []string. So either you put it back to a []string, or the other methods should accept a string as a command instead of []string. I'd prefer the first option to be honest.